doc/classes/KinematicBody2D.xml

```diff
@@ -63,7 +69,7 @@
 			<argument index="0" name="rel_vec" type="Vector2">
 			</argument>
 			<description>
-				Moves the body along the given vector. The body will stop if it collides. Returns a [KinematicCollision2D], which contains information about the colliding body.
+				Moves the body along the given vector. The body will stop if it collides. Returns a [KinematicCollision2D], which contains information about the collision.
```

Comment:
You can use the argument in the description for reference. Especially here as the name isn't crystal clear;
`Moves the body along the [code]rel_vec[/code] vector.`